tests/bcl-test/Mono.Data.Sqlite/Mono.Data.Sqlite-mac.csproj.template

```diff
@@ -53,7 +53,7 @@
     <Reference Include="Mono.Data.Sqlite" />
     <Reference Include="Xamarin.Mac" />
     <Reference Include="GuiUnit">
-      <HintPath>..\..\..\external\guiunit\src\framework\obj\Debug\net-4.5\GuiUnit.exe</HintPath>
+      <HintPath>..\..\..\external\guiunit\src\framework\obj\Debug\xammac_mobile\GuiUnit.exe</HintPath>
```

Comment:
Why is this xammac_mobile?

```
 $ cat ./System/System-mac.csproj.template | grep GuiUnit
    <Reference Include="GuiUnit">
      <HintPath>..\..\..\external\guiunit\src\framework\obj\Debug\net-4.5\GuiUnit.exe</HintPath>

 cat ./System.Xml/System.Xml-mac.csproj.template | grep GuiUnit
    <Reference Include="GuiUnit">
      <HintPath>..\..\..\external\guiunit\src\framework\obj\Debug\net-4.5\GuiUnit.exe</HintPath>

```

Reply:
Maybe this should be a project reference instead, that should solve the problem.